Public/Src/Utilities/Configuration/Mutable/SandboxConfiguration.cs

```diff
@@ -213,5 +215,8 @@ public UnsafeSandboxConfiguration UnsafeSandboxConfigurationMutable
 
         /// <inheritdoc/>
         ISandboxContainerConfiguration ISandboxConfiguration.ContainerConfiguration => ContainerConfiguration;
+
+        /// <inheritdoc />
+        public AdminRequiredProcessExecutionMode AdminRequiredProcessExecutionMode { get; set; }

```

Comment:
>AdminRequiredProcessExecutionMode [](start = 49, length = 33)

More proof that in the DScript side it should be an enum instead of a simple boolean

Reply:
See the discussion about enum in the previous comment.

---
In reply to: [276827371](https://github.com/Microsoft/BuildXL/pull/159#discussion_r276827371) [](ancestors = 276827371)